meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx_git.bb

```diff
@@ -16,7 +16,6 @@ SRC_URI = "${KERNEL_REPO};protocol=${KERNEL_REPO_PROTOCOL};branch=${KERNEL_BRANC
     file://0004-FIO-toup-hwrng-optee-support-generic-crypto.patch \
     file://0001-FIO-extras-arm64-dts-imx8mm-evk-use-imx8mm-evkb-for-.patch \
     file://0001-arm64-dts-imx8mq-drop-cpu-idle-states.patch \
-    file://0001-FIO-fromlist-gpu-drm-imx-sec_mipi_dsim-imx-fix-probe.patch \
```

Comment:
This patch is not used anywhere else, so the file `0001-FIO-fromlist-gpu-drm-imx-sec_mipi_dsim-imx-fix-probe.patch` can be removed too.

Reply:
ohhhhhhhhhh, great catch! Thanks